sway-core/src/ir_generation/function.rs

```diff
@@ -10,7 +10,7 @@ use crate::{
 use super::{compile::compile_function, convert::*, lexical_map::LexicalMap, types::*};
 
 use fuel_crypto::Hasher;
-use sway_ir::*;
+use sway_ir::{Context, *};
```

Comment:
What is the thinking here?  Trying to emphasise `Context` somehow?  Does it clash with the new `Context`?

Reply:
Yeah there's a glob import from both `sway_ir` and `semantic_analysis` here :smiling_face_with_tear: the explicit import takes precedence in Rust so this resolves the conflict.

Reply:
I didn't know you could do that.  Cool. :slightly_smiling_face: